web/react/stores/user_store.jsx

```diff
@@ -64,12 +65,12 @@ var UserStore = assign({}, EventEmitter.prototype, {
   },
   setCurrentId: function(id) {
     if (id == null)
-      sessionStorage.removeItem("current_user_id");
+      localStorage.removeItem("current_user_id");
```

Comment:
Yes that is a good idea. There is also a function in utils.jsx that should be in browser store. I will amend this PR with those updates.